test/functional/fixtures/kbn_archiver/discover.json

```diff
@@ -48,4 +48,34 @@
   ],
   "type": "search",
   "version": "WzUsMl0="
+}
+
+{
+  "attributes": {
+    "columns": [
+      "_source"
+    ],
+    "description": "A Saved Search With Empty Sort Description",
+    "hits": 0,
+    "kibanaSavedObjectMeta": {
+      "searchSourceJSON": "{\"highlightAll\":true,\"filter\":[],\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}},\"indexRefName\":\"kibanaSavedObjectMeta.searchSourceJSON.index\"}"
+    },
+    "sort": [[]],
```

Comment:
I wasn't sure of the best way to test for this, so I added this saved search to our functional tests which will cause them to fail to run if the empty sort array doesn't pass validation.